code/modules/defenses/sentry.dm

```diff
@@ -546,6 +546,45 @@
 	handheld_type = /obj/item/defenses/handheld/sentry/mini
 	composite_icon = FALSE
 
+/obj/structure/machinery/defenses/sentry/premade/deployable/colony/landing_zone
+	name = "\improper UA-577 Spaceborn Gauss Turret"
+	fire_delay = 2
+	omni_directional = TRUE
+	var/battery_low = 7 MINUTES
+	var/battery_critical = 9 MINUTES
+	var/battery_dead = 10 MINUTES
+	var/battery_state = TURRET_BATTERY_STATE_OK
+
+/obj/structure/machinery/defenses/sentry/premade/deployable/colony/landing_zone/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(set_battery_state), TURRET_BATTERY_STATE_LOW), battery_low)
+	addtimer(CALLBACK(src, PROC_REF(set_battery_state), TURRET_BATTERY_STATE_CRITICAL), battery_critical)
+	addtimer(CALLBACK(src, PROC_REF(set_battery_state), TURRET_BATTERY_STATE_DEAD), battery_dead)
+
+/obj/structure/machinery/defenses/sentry/premade/deployable/colony/landing_zone/get_examine_text(mob/user)
+	. = ..()
+	switch(battery_state)
+		if(TURRET_BATTERY_STATE_LOW)
+			. += "Its battery indictor is flashing yellow."
+		if(TURRET_BATTERY_STATE_CRITICAL)
+			. += "Its battery indictor is flashing red."
+		if(TURRET_BATTERY_STATE_DEAD)
+			. += "It appears to be offline."
```

Comment:
Why should it be omitted? 
```suggestion
/obj/structure/machinery/defenses/sentry/premade/deployable/colony/landing_zone/get_examine_text(mob/user)
	. = ..()
	switch(battery_state)
		if(TURRET_BATTERY_STATE_OK)
			. += "Its battery indictor is green, fully charged."
		if(TURRET_BATTERY_STATE_LOW)
			. += "Its battery indictor is flashing yellow."
		if(TURRET_BATTERY_STATE_CRITICAL)
			. += "Its battery indictor is flashing red."
		if(TURRET_BATTERY_STATE_DEAD)
			. += "It appears to be offline."
```

Reply:
Sure though its because its already a giant examine block so I didn't deem it necessary to include when its not a concern yet. 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/a89abd62-5c88-496d-b2b0-ed35a7b583d0)